docs/pages/architecture/agents.mdx

```diff
@@ -0,0 +1,280 @@
+---
+title: Teleport Agent Architecture
+description: Describes the architecture that enables Teleport to securely proxy client traffic to infrastructure resources. 
+tocDepth: 3
+---
+
+**Teleport agents** route traffic from users through the Teleport Proxy Service
+to and from resources in your infrastructure. This guide describes the
```

Comment:
```suggestion
**Teleport agents** route traffic from users through the Teleport Proxy Service
to and from resources in your infrastructure. This guide describes the
```

_Teleport agents route traffic from users through the Teleport Proxy Service_ seems to imply that the traffic path is user-->agent-->proxy. Are you reading it the same way or am I just being pedantic?

Reply:
I've made this, "**Teleport agents** route traffic from users to and from resources in your infrastructure". I think being more general is fine in the intro paragraph, and the guide clarifies the role of the Proxy Service in detail later.